### PR TITLE
fix(text-input): switched from outline to border on focus

### LINKED ISSUE #530: Text input in modal misses left outline on focus - IE/Firefox
Left outline is not visible when a text input is focussed inside a modal. The issue happens only in IE and firefox

image

**Temporary Workaround**
Adding a small left padding unhides the left outline
`.bx--modal-content .bx--form-item {
	padding-left:1px;
}`

### PR DESCRIPTION
**Pull request by marijohannessen**
## Overview

Closes #530 

Switched from using an outline to using border on focus for the text inputs to prevent the outline from being cut off in a modal in IE11/Firefox.

### Changed

- `_text-input.scss`

## Testing / Reviewing

Make sure that a text input is not cut off on focus anymore when used in a modal in IE11/Firefox.

### CODE REVIEW

File: src/components/modal/modal.html [hunk 1]
```diff
@@ -12,19 +12,17 @@
         </svg>
       </button>
     </div>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id accumsan augue. Phasellus consequat augue vitae tellus
+      tincidunt posuere. Curabitur justo urna, consectetur vel elit iaculis, ultrices condimentum risus. Nulla facilisi.
+      Etiam venenatis molestie tellus. Quisque consectetur non risus eu rutrum. </p>
+  </div>
 
-    <div class="bx--modal-content">
```
Comment (asudoh) [inline src/components/modal/modal.html]: Are we eliminating this `<div>`?

**Review by asudoh — COMMENTED** [review]
Thanks @marijohannessen for coming up with a quick fix! I’d suggest that we have a quick review with our designers, though I think you may have done this already. Thanks!

File: src/components/modal/modal.html (hunk 1 above)
Comment (alisonjoseph) [reply src/components/modal/modal.html]: Looks like the modal html file was edited in error? If I revert looks good in IE.

Comment (alisonjoseph) [reply src/components/modal/modal.html]: Agree, looks like this was edited in error?

**Review by alisonjoseph — CHANGES_REQUESTED** [review]
Looks good other than the edits to the modal html file. 😄

File: src/components/modal/modal.html (hunk 1 above)
Comment (marijohannessen) [reply src/components/modal/modal.html]: It certainly was 🙂

**Comment by marijohannessen** [comment]
It's been reviewed @asudoh !

**Review by alisonjoseph — APPROVED** [review]

**Review by asudoh — APPROVED** [review]
Thanks @marijohannessen for checking the change with our designers! 👍

**Pull request merged**